style.css

```diff
@@ -0,0 +1,103 @@
+.navbar {
```

Comment:
You should have brackets on new line. 
(e.g. 
`.class \n
{ \n
... \n
}`) "\n" denote new lines

Reply:
do i need to set my vs code for that

Reply:
"\n" is just a new line. If you see a \n it means press Enter on your keyboard